core/src/test/java/com/microsoft/applicationinsights/common/SanitizationUtilsTests.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.applicationinsights.common;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SanitizationUtilsTests {
+
+    @Test
+    public void testSanitizeStringForJSON() {
+        String result = SanitizationUtils.sanitizeStringForJSON("{\"entityType\":\"CONTACTNAME\",\"contactField\":\"\",\"query\":\"zuck\",\"candidates\":[{\"id\":\"2\",\"name\":\"Andrew Zuck\",\"aliases\":[],\"phoneNumbers\":[{\"type\":\"MOBILENUMBER\",\"number\":\"+1 (555) 222-2222\"}],\"emailAddresses\":[],\"streetAddresses\":[]}],\"scores\":{\"2\":2.327993297520463}}");
```

Comment:
This test does not seem to validate `\f`, `\b`, `\n`, `\r`, `\t`, `/`, `\` cases - please add another test case(s) to cover for this.